composeCore/src/main/java/com/hamdan/forzenbook/compose/core/composables/Composables.kt

```diff
@@ -297,12 +302,44 @@ fun ForzenbookTopAppBar(
         },
         actions = {
             if (actions != null) actions()
-            else Icon(
-                imageVector = Icons.Filled.ArrowBack,
-                contentDescription = stringResource(id = R.string.back_arrow),
-                tint = Color.Transparent,
+        }
+    )
+}
+
+@Composable
+fun ForzenbookBottomNavigationBar(navIcons: List<NavigationItem>) {
+    BottomAppBar(
+        containerColor = ForzenbookTheme.colors.colors.background
+    ) {
+        navIcons.forEach {
+            NavigationBarItem(
+                selected = false,
+                onClick = { Log.v("Hamdan", it.page) },
+                label = { Text(text = stringResource(id = it.label)) },
+                icon = {
+                    Image(
+                        painterResource(id = it.icon),
+                        contentDescription = stringResource(id = it.description),
+                        colorFilter = ColorFilter.tint(ForzenbookTheme.colors.colors.primary)
```

Comment:
I don't believe you need to wrap this in that tint function if you use `Icon` instead of `Image`?

Reply:
yes you are correct.